src/meadowflow/jobs.py

```diff
@@ -371,9 +375,10 @@ async def execute(
             # TODO meadowflow needs to store credentials the same way that the
             #  meadowgrid coordinator does, or share them, or something. Currently
             #  we are passing None because we don't have any.
-            credentials_dict = {}
+            credentials_dict: CredentialsDict = {}
 
             # convert a job_function into a job_runner_function
+            job_runner_function: Union[LocalFunction, MeadowGridDeployedRunnable]
```

Comment:
Probably better to call this a `JobRunnerFunction`?

Reply:
didn't get this one, this is a variable not a type?

Reply:
I think `Union[LocalFunction, MeadowGridDeployedRunnable] == JobRunnerFunction`

Reply:
oh, right, good catch